sql/core/benchmarks/DateTimeRebaseBenchmark-jdk11-results.txt

```diff
@@ -59,36 +59,36 @@ OpenJDK 64-Bit Server VM 11.0.6+10-post-Ubuntu-1ubuntu118.04.1 on Linux 4.15.0-1
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 Save dates to ORC:                        Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-after 1582, noop                                   9295           9295           0         10.8          93.0       1.0X
-before 1582, noop                                  9352           9352           0         10.7          93.5       1.0X
-after 1582                                        17112          17112           0          5.8         171.1       0.5X
-before 1582                                       17979          17979           0          5.6         179.8       0.5X
+after 1582, noop                                   9250           9250           0         10.8          92.5       1.0X
+before 1582, noop                                  9522           9522           0         10.5          95.2       1.0X
+after 1582                                        16377          16377           0          6.1         163.8       0.6X
```

Comment:
@cloud-fan I don't think it worth to optimize the write path. The code looks ugly, from my point of view.